src/gui/main.c

```diff
@@ -338,7 +338,16 @@ int main(int argc, char *argv[])
       dt_gui_present();
 
     static int bs = 1;
-    if(bs) { dt_gui_toggle_fullscreen(); bs = 0; }
+    if (bs) {
```

Comment:
uhm i don't understand this part. isn't it just a verbatim copy of the toggle_fullscreen call that has been removed? up to setting the x and y coordinate too? if so, why not change the offsets in api.h:dt_gui_toggle_fullscreen directly?